MyApp.Shared/Components/Pages/Counter.razor

```diff
@@ -1,4 +1,5 @@
 ﻿@page "/counter"
+@rendermode @(GlobalRenderSettings.GlobalRenderMode)
```

Comment:
Not sure if you think it would be better, but if you were to add:

```
@using static GlobalRenderSettings
```

into `_Imports.razor`, then could this be simplified to:

```
@rendermode GlobalRenderMode
```

If so that would feel pretty natural.

Reply:
Yeah I think the specific code/names in here are just a placeholder. We definitely need better names.

Reply:
I was more focused on simplifying `@rendermode @(X.Y)` to just `@rendermode Y`

Reply:
Oh got it. If that compiles I'll do it!

Reply:
Works great, much cleaner!